### PR TITLE
Minor doc fixes

### PR DESCRIPTION
**Pull request by jovatn**
## Description

Adding/fixing some cross links, moving some picture, typos etc

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Test changes
  - [ ] Checked in changed Readme.adoc (make test-spec)
  - [ ] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [X] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

File: doc/boot.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ which offers lots of benefits out-of-the-box:
   media like USB sticks or SD-cards) and from remote servers using FTP
   or HTTP(S).
 
-To initiate a system upgrade, run:
+To initiate a system upgrade from shell[^1], run:
```
Comment (troglobit) [inline doc/boot.md:146]: from _the_ shell?

File: doc/boot.md [hunk 2]
```diff
@@ -158,7 +158,6 @@ same upgrade command again will bring both partitions into sync.
 
 [RAUC]: https://rauc.io
 
-
```
Comment (troglobit) [inline doc/boot.md:161]: Style thing, I try to keep double linefeeds between sections.

**Review by troglobit — CHANGES_REQUESTED** [review]
Nice work! Minor comments only.

File: doc/boot.md (hunk 2 above)
Comment (jovatn) [reply doc/boot.md:161]: Oops, thx!

**Review by troglobit — APPROVED** [review]

**Pull request merged**